near-sdk/src/collections/legacy_tree_map.rs

```diff
@@ -585,6 +585,16 @@ where
         self.save(&n);
         self.tree.pop();
     }
+
+    pub fn to_tree_map<S: IntoStorageKey>(self, prefix: S) -> super::TreeMap<K, V> {
```

Comment:
```suggestion
    pub fn into_tree_map<S: IntoStorageKey>(self, prefix: S) -> super::TreeMap<K, V> {
```

Just to follow https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv naming conventions.

Also, would be nice to have a small comment describing this fn